core/server/worker/src/main/java/alluxio/worker/file/FileDataManager.java

```diff
@@ -89,6 +93,7 @@ public FileDataManager(BlockWorker blockWorker, RateLimiter persistenceRateLimit
     mPersistedUfsFingerprints = new HashMap<>();
     mPersistenceRateLimiter = persistenceRateLimiter;
     mUfsManager = ufsManager;
+    mUfsSessionManager = new WorkerUfsSessionManager(ufsManager);
```

Comment:
I'll change this to a single session manager per service if the overall design makes sense